### DISCUSSION TITLE
Minimising windows

### DISCUSSION THREAD

**Discussion by neubami94** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#47] [Ideas]
Hi, thanks for this amazing app. But could you also add the feature, where you hover over the minimizing button in the top right corner while dragging to minimze it? I am sure it behaved like that in win10 :(

**Comment by HerMajestyDrMona** [comment]
No, it didn't work this way in Windows 10, I just tested on a Virtual Machine.

And even if it worked this way on Windows 10, it would be very hard (or maybe impossible) to add, because it would require "sniffing" of all active/foreground windows, suspecting the minimize button position, etc. Some windows have custom buttons layout. That wouldn't be something worth focusing at all.

What you might remember from Windows 10 is dragging to the "Show Desktop" area (bottom-right corner of the screen), and it already works this way with this program too.
